modules/chapter2/pages/section3.adoc

```diff
@@ -0,0 +1,184 @@
+For S3 storage, this can be done with the `boto3` library, as you will learn in the following exercise.
+
+After you are connected to S3, then you can start downloading the files required for data exploration or model training.
+
+Similarly, you should store your trained models in the S3 that corresponds to the data connection.
+In fact, storing your models in the data connection enables you to use Model Serving.
+Model Serving downloads the model files from S3 by using a data connection.
+
+
+
+
+
+
+== Exercise: Getting Started with Data Connections
+
+=== Create a Workbench with a Data connection
+
+From a data science project page, click btn:[Create Workbench].
+
+Select the btn `Standard Data Science` notebook image for this workbench.
+
+[IMPORTANT]
+====
+Make sure you use the `Standard Data Science` image.
+This image contains the `boto3` library, required for the second part of the exercise.
+====
+
+[NOTE]
+====
+Alternatively, you can add a data connection to an existing workbench by editing the workbench.
+====
+
+Scroll down to the btn:[Data connections] section and select btn:[Create data connection].
+This option creates the data connection and assigns it to the workbench that you are creating.
+Complete the form with your S3 connection parameters.
+
+image::dataconnection-new.png[]
+
+Save the workbench.
+In the data science project page, notice the newly created data connection, which is associated with the workbench.
+
+image::dataconnection-list.png[]
+
+[NOTE]
+====
+The association between data connections and workbenches follows a `1 to N` schema:
+
+* The same data connection can be used in multiple workbenches.
+* A workbench can have only one data connection.
+====
+
+=== Edit a Data connection
+
+Next, click the btn:[⋮] button of the data connection, then click btn:[Edit data connection].
+Note that you can assign more workbenches to the same data connection.
+
+image::dataconnection-edit.png[]
+
+You do not need to make any changes.
+
+[NOTE]
+====
+You can use the data science project page to create new data connections and assign them to existing workspaces.
+
+You can also use this page to delete data connections.
+Deleting a data connection that is assigned to a workbench results in a workbench restart.
+====
+
+=== Using the Data Connection in a Workbench
+
+After you have created the data connection and assigned it to your workbench, follow these steps:
+
+1. *Clone the demo code.*
+
+a. Open the workbench JuypterLab URL.
+
+b. If prompted, log in with your Red{nbsp}Hat OpenShift credentials.
+
+c. Click btn:[Allow selected permissions] to grant the workbench access to your data science project.
+
+d. Click the btn:[Git] icon in the left sidebar of JupyterLab.
+
+e. Click btn:[Clone a repository].
++
+image::git-clone-menu.png[width=40%,align="center"]
+
+f. Enter https://github.com/RedHatQuickCourses/rhods-intro.git as the repository, and click btn:[Clone].
```

Comment:
Hmmm. This will download the entire adoc source tree. Can we think about storing all notebook and lab code in a separate top-level repo in common with all the 6 quick courses?